homeassistant/components/met/config_flow.py

```diff
@@ -66,6 +78,9 @@ async def _show_config_form(
                     vol.Required(CONF_LATITUDE, default=latitude): cv.latitude,
                     vol.Required(CONF_LONGITUDE, default=longitude): cv.longitude,
                     vol.Required(CONF_ELEVATION, default=elevation): int,
+                    vol.Required(CONF_MODE, default=DEFAULT_MODE): vol.In(
```

Comment:
The mode option should be optional since we have a default and be moved to an options flow. We only want required options in the main flow.

https://developers.home-assistant.io/docs/config_entries_options_flow_handler